cli/module_loader.rs

```diff
@@ -70,6 +70,24 @@ impl CliModuleLoader {
         found_url: specifier.to_owned(),
         media_type: MediaType::JavaScript,
       });
+    } else if specifier.as_str() == "node:process" {
+      return Ok(ModuleCodeSource {
+        code: deno_runtime::deno_node::PROCESS_ES_SHIM.to_string(),
+        found_url: specifier.to_owned(),
+        media_type: MediaType::JavaScript,
+      });
+    } else if specifier.as_str() == "node:os" {
+      return Ok(ModuleCodeSource {
+        code: deno_runtime::deno_node::OS_ES_SHIM.to_string(),
+        found_url: specifier.to_owned(),
+        media_type: MediaType::JavaScript,
+      });
+    } else if specifier.as_str() == "node:tty" {
+      return Ok(ModuleCodeSource {
+        code: deno_runtime::deno_node::TTY_ES_SHIM.to_string(),
+        found_url: specifier.to_owned(),
+        media_type: MediaType::JavaScript,
+      });
```

Comment:
Do we also need to add `node:events` clause? I bumped into the error below when trying to print out `vite.middlewares` using Deno 1.29.1.

<img width="811" alt="Screenshot 2023-01-08 at 5 58 55 PM" src="https://user-images.githubusercontent.com/242017/211190040-456aa5a5-c8cf-4ac5-9ffc-f31206480639.png">

Reply:
The issue you are running in will be resolved by https://github.com/denoland/deno/pull/17264

Reply:
Oh ok thanks.